ddl/index.go

```diff
@@ -1173,10 +1160,10 @@ func (w *addIndexWorker) handleBackfillTask(d *ddlCtx, task *reorgIndexTask) *ad
 	return result
 }
 
-func (w *addIndexWorker) run(d *ddlCtx) {
+func (w *backfillWorker) run(d *ddlCtx, bf backfiller) {
 	logutil.BgLogger().Info("[ddl] add index worker start", zap.Int("workerID", w.id))
 	defer func() {
-		w.resultCh <- &addIndexResult{err: errReorgPanic}
+		w.resultCh <- &backfillResult{err: errReorgPanic}
```

Comment:
The log information in this function seems to be outdated.

Reply:
Ditto.